crates/ra_ide/src/hover.rs

```diff
@@ -13,14 +13,43 @@ use ra_ide_db::{
 use ra_syntax::{ast, match_ast, AstNode, SyntaxKind::*, SyntaxToken, TokenAtOffset};
 
 use crate::{
-    display::{macro_label, rust_code_markup, rust_code_markup_with_doc, ShortLabel},
-    FilePosition, RangeInfo,
+    display::{macro_label, rust_code_markup, rust_code_markup_with_doc, ShortLabel, ToNav},
+    FilePosition, NavigationTarget, RangeInfo,
 };
 
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct HoverConfig {
+    pub implementations: bool,
+}
+
+impl Default for HoverConfig {
+    fn default() -> Self {
+        Self { implementations: true }
+    }
+}
```

Comment:
Since the client has to opt in to this feature it makes more sense for the default to be false.

Reply:
Initially it was. 
> matklad said(https://github.com/rust-analyzer/rust-analyzer/pull/4659#discussion_r433974798)
>>I think in ra_ide we generally enable everything by deafault, and than rust-analyzer explicitelly disables stuff it doesn't like.